cockatrice/src/gameselector.cpp

```diff
@@ -290,13 +292,15 @@ void GameSelector::actSelectedGameChanged(const QModelIndex &current, const QMod
     joinButton->setEnabled(game.player_count() < game.max_players() || overrideRestrictions);
 }
 
-void GameSelector::setAlteredFiltersText(const int numAlteredFilters)
+void GameSelector::setFilteredGamesLabel()
 {
-    if (alteredFiltersLabel != nullptr) {
-        if (numAlteredFilters == 0) {
-            alteredFiltersLabel->setText(tr("Default filters applied"));
-        } else {
-            alteredFiltersLabel->setText(tr("%1 filter(s) applied").arg(numAlteredFilters));
-        }
+    if (filteredGamesLabel != nullptr && gameListProxyModel != nullptr &&
+        gameListModel != nullptr) {
+        const int totalGames = gameListModel->rowCount();
+        const int shownGames = totalGames - gameListProxyModel->getNumFilteredGames();
+        filteredGamesLabel->setText(tr("Games shown: %1 / %2").arg(shownGames).arg(totalGames));
+    } else {
+        filteredGamesLabel->setText(tr("Games shown: NaN / NaN"));
```

Comment:
Wouldn't show a weird error to a user like this. Would silently fail

Reply:
Oh sure, that was more for my own debugging :) is there somewhere we'd want to log an error, or is this trivial enough to not be useful?

Reply:
I find it trivial, but you can dump to qDebug() so we can trace it later if needed

Reply:
(Removing this check per other discussion)